config/loader_test.go

```diff
@@ -1,18 +1,23 @@
 package config
 
-import "testing"
+import (
+	"os"
+	"testing"
+)
 
 func TestString(t *testing.T) {
+	os.Setenv("TESTVAR", "test")
```

Comment:
This is basically a global variable and should be avoided. Can you create a loader in each test with environment variables as required?